Update migration-migrate-delta.md (#3615)
magento/devdocs: guides/v2.1/migration/migration-migrate-delta.md

* all operations with orders in the {% glossarytooltip 18b930cf-09cc-47c9-a5e5-905f86c43f81 %}Magento Admin{% endglossarytooltip %} panel

> * !The Tool will not migrate new products or categories. These must be migrated manually

Reviewer: ```suggestion
* !The Tool will not migrate new products or categories. These must be migrated manually.
```

Reviewer 2: ```suggestion
{: .bs-callout .bs-callout-warning }
All the other new or updated in Admin Panel entities like products or categories are not included into incremental migration and wont be migrated. 
```

Reviewer 2: I do not know how symbol "!" will decor the text but it may be better to make it as {: .bs-callout .bs-callout-info } then another bullet. And in the text is better to say that not only products or categories will not be migrated but everything else

Author: I am fine with this change

Reviewer 3: @victor-v-rad is correct on the formatting for the note. The ! in kramdown won't add a note. But having something like this will put it in a blue info box. If a warning, change info to warning in the first line.

```
{: .bs-callout .bs-callout-info }
All new or updated entities entered through the Admin panel like products or categories are not included into incremental migration and will not be migrated. 
```

Reviewer 2: @jeff-matthews 
The last update from @lorikrell looks good to me

* all operations with orders in the {% glossarytooltip 18b930cf-09cc-47c9-a5e5-905f86c43f81 %}Magento Admin{% endglossarytooltip %} panel

{: .bs-callout .bs-callout-info }

> All new or updated entities entered through the Admin panel like products or categories are not included into incremental migration and will not be migrated.

Reviewer: ```suggestion
All new or updated entities entered through the Admin panel, like products and categories, are not included in incremental migration and will not be migrated.
```
